descriptions/sentieon_merge_TNfilter_normal.cwl

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+
+class: CommandLineTool
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+  - class: EnvVarRequirement
+    envDef:
+      -
+        envName: SENTIEON_LICENSE
+        envValue: LICENSEID
+
+hints:
+  - class: DockerRequirement
+    dockerPull: ACCOUNT/sentieon:VERSION
+
+baseCommand: [sentieon_merge_TNfilter_normal.sh]
+
+inputs:
+  - id: genome_reference_fasta
+    type: File
+    secondaryFiles:
+      - ^.dict
+      - .fai
+    inputBinding:
+      position: 1
+    doc: Genome reference in FASTA format with the corresponding index files
+
+  - id: sample_name
+    type: string
+    inputBinding:
+      position: 2
+    doc: Name of the sample
+
+  - id: normal_name
```

Comment:
this variable name is a bit confusing, maybe `normal_sample_name` would be more clear